assets/react/v3/shared/components/fields/FormCategoriesInput.tsx

```diff
@@ -118,13 +120,19 @@ const FormMultiLevelInput = ({
                 <Controller
                   name="name"
                   control={form.control}
+                  rules={{
+                    required: __('Category name is required', 'tutor'),
+                  }}
                   render={(controllerProps) => (
                     <FormInput {...controllerProps} placeholder={__('Category name', 'tutor')} selectOnFocus />
                   )}
                 />
                 <Controller
                   name="parent"
                   control={form.control}
+                  rules={{
```

Comment:
Parent is not required. Send `parent` only when it's selected

Reply:
Resolved